lib/status_code/locales/en.yml

```diff
@@ -403,6 +403,390 @@ en:
       '502': Decline, pick-up card, card issuer decline reason # Отказ от эмитента – необходим захват карты
       '809': Error, system error # Системная ошибка
       '888': Error, invalid session number # Неверный номер сессии
+    bank_one:
```

Comment:
We should use `bankone` in current circumstances, because of the [following](https://github.com/beGateway/status_code/blob/master/lib/status_code.rb#L13)
I propose:
Force the user of the library to send already prepared gateway and do nothing with the gateway name [here](https://github.com/beGateway/status_code/blob/master/lib/status_code.rb#L13)